SoS/SoS.tp2

```diff
@@ -1082,15 +1090,21 @@ BEGIN @21011
                 ~%MOD_FOLDER%/Pvrz/Alt/A4230N17.PVRZ~ ~override~
                 ~%MOD_FOLDER%/Pvrz/Alt/A4230N18.PVRZ~ ~override~
                 ~%MOD_FOLDER%/Pvrz/Alt/A4230N19.PVRZ~ ~override~
+                ~%MOD_FOLDER%/Mos/ALT/ee/AR4230.MOS~ ~override~
+                ~%MOD_FOLDER%/Mos/ALT/ee/AR4230N.MOS~ ~override~
     END
     ELSE BEGIN
-        COPY_LARGE    ~%MOD_FOLDER%/Tiz/ALT/AR4230.TIS~ ~override~
-                    ~%MOD_FOLDER%/Tiz/ALT/AR4230N.TIS~ ~override~
+        INCLUDE ~%MOD_FOLDER%/lib/a7_tools.tpa~
```

Comment:
It seems that this file is already included in line 1016

Reply:
@Roberciiik Yes but it's for the previous component.

Reply:
It could be added to ALWAYS phase then. But this is fine too.